README.md

```diff
@@ -38,207 +12,105 @@ Add this line to your application's Gemfile:
-keys.
-
-The I18n library supports 'interpolation' using the following syntax:
-
-```yaml
-en:
-  some_view:
-    greeting: Hello, %{name}
-```
-
-in this case the application can pass in the value of the `name` variable.
-
-If a translation includes an interpolation placeholder which has not been
-given a value by the application backend, then a runtime error will be raised.
-
-Unfortunately the placeholder variables sometimes get changed by mistake, or
-by translators who are not aware that they should not modify text within the
-special curly braces of the interpolation placeholder.
-
-This is obviously not great, and the validation task is intended to guard
-against it.
-
-It will check all translation files and report any which contain placeholders
-which do not exist in the english file.
-
-```
-$ rake translation:validate
-Success! No unexpected interpolation keys found.
-```
+After a `bundle install`, this will make a number of rake tasks available to your application.
+See the [Rake command reference](#rake-command-reference) below.
 
-### Stealing translations from another app
+You will now also be able to run tests against your locale files to ensure that they are valid.
+Create a test file as follows.
 
-A third feature is the ability to "steal" one or more locales from an existing
-application. This functionality works by providing a mapping file, which defines
-how the translation keys in the the source app's files map to those in the app
-the gem is installed in.
+#### Minitest
 
-For example, given a locale file like this in the app to "steal" from:
-
-```yaml
-es:
-  document:
-    type:
-      case_study: Caso de estudio
-      consultation: Consulta
+```ruby
+class LocalesValidationTest < ActiveSupport::TestCase
+  test "should validate all locale files" do
+    checker = RailsTranslationManager::LocaleChecker.new("config/locales/*.yml")
+    assert checker.validate_locales
+  end
+end
 ```
 
-and a mapping file like this:
+#### RSpec
 
-```yaml
-document.type: content_item.format
+```ruby
+RSpec.describe "locales files" do
+  it "should meet all locale validation requirements" do
+    checker = RailsTranslationManager::LocaleChecker.new("config/locales/*/*.yml")
+    expect(checker.validate_locales).to be_truthy
+  end
+end
 ```
 
-running `rake translation:steal[es,../other_app,mapping_file_path.yml]` will
-result in the following locale file being created:
-
-```yaml
-es:
-  content_item:
-    format:
-      case_study: Caso de estudio
-      consultation: Consulta
-```
+### Running the test suite
 
-The mapping file can live anywhere, as long as the full path (including filename)
-is given in the rake task invocation.
+To run the test suite just run `bundle exec rake` from within the
+`rails_translation_manager` directory.
 
-The process will preserve data already in the output file if it is not
-referenced in the mapping, but will always override data belonging to keys
-that are in the mapping.
+### Further documentation
```

Comment:
For some reason the markdown isn't being rendered here 🤔 https://github.com/alphagov/rails_translation_manager/blob/2fb824afd8eb8afb1380eaaef053f9175de07b85/README.md

Reply:
Ah, the old accidental `ALT + SPACE` OSX issue - happens a lot! Good spot 👍